### ISSUE TITLE
Refresh opensim-moco conda packages

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/conda-opensim#27]
Issue reported on user forum 
https://simtk.org/plugins/phpBB/posting.php?mode=reply&f=91&t=15999&sid=4c36d5d568641f0b859cef704000c9c9
opensim-moco package on osx is stale, and on linux doesn't exist. 

This will be necessary to run moco solutions using jupyter notebooks

**Comment by nickbianco** [comment]
There is recent interest in adding opensim-moco packages for Linux on the Moco forum: https://simtk.org/plugins/phpBB/viewtopicPhpbb.php?f=91&t=17353&p=48260.

@aymanhab, perhaps we could make this a priority for upcoming dev sprints?

**Comment by aymanhab** [comment]
Indeed, the bottleneck though seems to be the older/stale ipopt configuration files not supporting later compilers. We should upgrade these versions instead of trying to trick/workaround the old code base.

**Comment by aymanhab** [comment]
This is now obsolete, will close to cleanup
